### ISSUE TITLE
Sprint 1.35.0 2/2 - Apr 18 to Apr 29

### ISSUE THREAD

**Issue by marcushyett-ph** [PostHog/posthog#9402]
# Global Sprint Planning

## Retro: Status of Outcomes from Previous Sprint

1. **Ingestion:** Have an MVP ready for persons data on new events (completed up to step 6 of [the project](https://github.com/orgs/PostHog/projects/41/views/1?visibleFields=%5B%22Title%22%2C%22Assignees%22%2C2635782%2C%22Status%22%2C2619643%2C2619894%5D)) @yakkomajuri (Status: Strongly blocked by upgrade to new clickhouse cluster)
2. **Team East: Nail Insights** https://github.com/PostHog/product-internal/pull/290 - @mariusandra @pauldambra (Status: progress on many-to-many, various smaller insight fixes, but will continue to next sprint)
3. **Team East: ~Nail~ Great Demo Data** https://github.com/PostHog/product-internal/pull/298 - @Twixes (Status [RFC done](https://github.com/PostHog/product-internal/pull/298) BUT implementation will go into the next sprint, as world map and insights enhancements took over as priorities)
3. **Team West: Cohort filtering** https://github.com/PostHog/product-internal/pull/286/files @EDsCODE (Status: Designs fully done, no implementation as design wasn't done + eric support)
4. **Infrastructure:** ClickHouse upgraded to 21.11.11.1.111.1, provision a new cluster and migrate over @guidoiaquinti  (Status: Scoped all work. Ground work required to set up lots of this stuff (networking etc). Quite a bit of work left to do)

## Plan: Proposed Goals for Next Sprint 

Each goal should have a single owner. Owner can only be an engineer.

1. Unblock persons on events querying on self hosted (@yakkomajuri)
  - why: To speed up querying
2. Have all infrastructure ready to do clickhouse migration (@guidoiaquinti)
  - why: Because we're blocked on doing schema migration 
3. Cohort filtering implementation (@EDsCODE)
  - why: lifecycle capabilities for cohorts are requested often as features
4. Get statsig increase in conversion from signup to first event ingested on onboarding experiment (@liyiy)
  - why: so we can have more customers be successful with PostHog
7. Have great demo data live on demo.posthog.com (@Twixes)
  - why: to increase sign ups
9.  Complete nail insights RFC implementations https://github.com/PostHog/product-internal/pull/290 (@mariusandra)
  - why: m2m dashboard: customer requests

# Team sprint planning

For your team sprint planning copy this template into a comment below for each team.

```
Team ___

## Retro

<!-- Talk about what went well, what didn't go well and any actions to improve next time -->

- 

## Hang over items from previous sprint

<!-- For each item, decide to re-prioritise (and add below) or deprioritise -->

- Item 1. prioritised/deprioritise

## Planning

<!-- Each item should have a single owner. Owner can only be an engineer. -->

### High priority

-

### Low priority / side quests

-

```

**Comment by mariusandra** [comment]
Team App East

@Twixes support first week

## Retro

- @mariusandra 1 week support, 1 week off, now doing things, but won't finish the plan
- @pauldambra off this week, kicked off insight many-to-many refactor last week
- @Twixes world map 🌐 🎉  (please edit)

## Hang over items from previous sprint

Basically everything

- [Nail Insights](https://github.com/PostHog/product-internal/pull/290) - @mariusandra @pauldambra (many-to-many, insight history, UX componentization)
- [Nail Great Demo Data](https://github.com/PostHog/product-internal/pull/298) - @Twixes (RFC seems solid, implementation follows)

## Planning

See above.

### High priority

See above.

### Low priority / side quests

- Frontend plugins/apps? How to get squeak into the posthog submenu?
- Toolbar revamp
- General Artificial Intelligence

**Comment by yakkomajuri** [comment]
# Team Ingestion

## Retro

- @yakkomajuri
    - Burned twice by unknown unknowns (Kafka ingestion + Zookeeper metadata), a lot of draining firefighting
    - Otherwise: good progress, **collaboration**, and planning
- @tiina303 
    - Good collaboration and planning
    - GitHub projects are working very well for us
    - We should plan better for hackathon overflows
        - @fuziontech suggestion: hackathons as "sprint plannings" 

## Hang over items from previous sprint

- Schema update (blocked by cluster issues with Zookeeper metadata)

## Planning

### Top priority

- Unblock persons on events querying on self hosted 

### Others

https://github.com/orgs/PostHog/projects/41
